.github/workflows/test-release.yml

```diff
@@ -14,4 +17,4 @@ jobs:
             - name: Run and set up the parachain, oracle, staked relayer and vault
               run: docker-compose up -d
             - run: yarn install
-            - run: yarn ci:test
+            - run: yarn ci:test:release
```

Comment:
This should run all tests on master, so just 'yarn test' I think

Reply:
Since the staging tests passed in the PR, I don't think we need to re-run them on `master`

Reply:
following scenario I have in mind:
PR1 changes staging tests X
PR2 changes staging tests Y

Both are passing. But there is an edge case where PR2 affects tests in PR1 that have just changed. Now, independently they pass, but on master they could fail. Example could be when we change the way that issue fees are deducted. In PR1, we could have fixed the issue tests. PR2 could be update of the vault struct. But maybe in the PR2, we added a new test that gets all the vault structs with the issuable tokens of the vault. But now that test would be incorrect because of PR1.

It's best practice to run all tests before a release to cover all of these edge cases.